presto-docs/src/main/sphinx/release/release-0.260.rst

```diff
@@ -0,0 +1,36 @@
+=============
+Release 0.260
+=============
+
+**Highlights**
+==============
+
+**Details**
+===========
+
+General Changes
+_______________
+* Fix a bug in sql functions that would cause compiler error when session property ``inline_sql_function`` is set to ``false`` and sql function references input in lambda expression.
+* Fix multithread writing for fragment result cache.
+* Add UUID type and operators.
```

Comment:
Add ``UUID`` type and operators.

Reply:
```
Add ``UUID`` type. See :doc:`types`
```

The fact that we added operators like `LIKE`, `CAST` is not that important (rather expected I'd guess) - so we can skip those.